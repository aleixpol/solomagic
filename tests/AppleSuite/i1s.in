\ref something0
\mt somethingsomething inaha somethingsomething

\ref something1
\mt somethingsomething tinaha somethingsomething

