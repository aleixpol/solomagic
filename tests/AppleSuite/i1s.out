\ref something0
\mt somethingsomething inaha somethingsomething
\ma somethingsomething ingnaha somethingsomething

\ref something1
\mt somethingsomething tinaha somethingsomething
\ma somethingsomething tinaha somethingsomething
